api/views/user_views.py

```diff
@@ -249,9 +249,9 @@ class UploadAudioView(APIView):
     @transaction.atomic
     def post(self, request, *args, **kwargs):
         instructor = get_object_or_404(Instructor, user=request.user)
-
+        title = request.data["title"]
```

Comment:
This line will probably break the they system if we deploy this PR without/before updating the front-end; could you please the necessary changes (one would probably be to use `get`) and make sure that deploying this PR won't break the system?

Reply:
Working on the front-end changes now, I'll update this PR when those changes are complete.